homeassistant/components/wemo/__init__.py

```diff
@@ -125,7 +125,7 @@ async def async_stop_wemo(event: Event) -> None:
         hass.bus.async_listen_once(EVENT_HOMEASSISTANT_STOP, async_stop_wemo)
     )
 
-    # Need to do this at least once in case statics are defined and discovery is disabled
+    # Need to do this at least once in case statistics are defined and discovery is disabled
```

Comment:
This should be `statics`. It's meant to refer to the 'statically' configured devices in configuration.yaml.